src/Events/QR/Redirections.php

```diff
@@ -47,7 +46,7 @@ public function unregister(): void {
 	 * @return string The fallback URL.
 	 */
 	public function get_fallback_url(): string {
-		return (string) ( tribe_get_option( Settings::get_option_slugs()['fallback'] ) ?: home_url() );
+		return (string) tribe_get_events_link();
```

Comment:
we might need to use `tribe_events_get_url()` that is the old way of fetching it.